.golangci.yml

```diff
@@ -51,6 +51,11 @@ linters-settings:
   gosimple:
     go: "1.18"
     checks: ["all"]
+  gci:
+    sections:
+      - standard
+      - default
+      - prefix(github.com/FerretDB/FerretDB)
```

Comment:
We have that section above already (merged in another PR)

Reply:
Going to drop this commit.